.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,57 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+---
+name: Problem Report
+about: Create a Report to help us improve
+---
+
+<!-- Thanks for reporting a problem for this project. READ THIS FIRST:
+
+This issue template is meant to REPORT Labrador software PROBLEMS ONLY
+
+Please DO NOT OPEN AN ISSUE:
+  - If your Labrador version is not the latest from the master branch, please update your device before submitting your issue. Your problem might already be solved. 
+The latest pre-release binaries of Labrador can be downloaded from GitHub Actions.
+
+Please take a few minutes to complete the requested information below. Our ability to provide assistance is greatly hampered without it. The details requested potentially affect which options to pursue. The small amount of time you spend completing the template will also help the volunteers providing the assistance to you to reduce the time required to help you.
+
+DO NOT DELETE ANY TEXT from this template! Otherwise the issue will be auto-closed.
+-->
+
+### PROBLEM DESCRIPTION
+_A clear and concise description of what the problem is._
+
+
+### REQUESTED INFORMATION
+_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_
+
+
+- [ ] Searched the problem in [issues](https://github.com/espotek-org/Labrador/issues)
+- [ ] Searched the problem in [discussions](https://github.com/espotek-org/Labrador/discussions)
+- [ ] Searched the problem in the [docs](https://github.com/espotek-org/Labrador/wiki)
+
+
+### TO REPRODUCE
+_Steps to reproduce the behavior:_
+
+
+### EXPECTED BEHAVIOUR
+_A clear and concise description of what you expected to happen._
+
+
+### SCREENSHOTS
+_If applicable, add screenshots to help explain your problem._
+
+
+### ADDITIONAL CONTEXT
+_Add any other context about the problem here._
+
+
+**(Please, remember to close the issue when the problem has been addressed)**
```

Comment:
I'm not sure this bit is necessary either.  We can close the issues ourselves once the answer has been provided.

Reply:
> We can close the issues ourselves once the answer has been provided.

Usually with a service-orriented approach, I would prefer the author to decide if an answer was really adressing the original intent and not the person that provided the answer.
Reasoning: There are many hurdles to overcome to understand original intent (language, background, etc) hence only author knows the real intent.

But an automated workflow can kick in after a period of time has passed to auto-close "stale" issues. Creating that workflow was on my internal ToDo list already.

@EspoTek Would that resolve the concern?

Reply:
I guess my main concern was that the person who just needs help is now being given half a dozen different rules to follow.  Again, might be a cultural thing.

Reply:
ok, I think we have a fundamental different approach about "how to get help".
My approach is:

1. open a discussion to get help
2. only if the discussion clarifies that the discussed topic is caused by a bug or requests a new feature
3. open new issue or feature request. 

These are from my view the rules that make support and development effective and don't create too much of a burden for users

Does that make sense?

Reply:
I agree with your approach, but I just think large amounts of text and imperative instructions to follow come across as a bit hostile.  The checklist in place now looks great IMO.